sdk/communication/Azure.Communication.Email/src/Generated/Models/ErrorDetail.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.Communication.Email
+{
+    /// <summary> The error detail. </summary>
+    public partial class ErrorDetail
```

Comment:
Is there any reason we aren't using the standard [ResponseError](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResponseError.cs)?